units/5-javascript1_1/sessions/1-variable/README.md

```diff
@@ -1,235 +0,0 @@
-#Lesson 1 - Variables
-
-![image](http://i.imgur.com/xwrZiJd.jpg)
-
-## Before Class
-
-### Objective
-
-Students will able to create, instantiate, and edit variables in JavaScript.
-
-### Key Points
-
-* JavaScript is a programming language that we can run in our browser.
-* Variables are like boxes; they store something.
-* Variables have a name and a value.
-* We can perform various operations on variables.
-* Each variable has a type.
-
-### Assessment
-
-Students will show progress toward reaching the objective based on their performance on the exit ticket quiz.
-
-### Vocabulary
-
-* JavaScript
-* Variable
-* Instantiate
```

Comment:
“Instantiate” is typically used to describe the process of creating a new instance of a class or prototype—it doesn’t have any relation to the material in this lesson.